glade_forms/lib/src/core/input/glade_input.dart

```diff
@@ -107,10 +107,12 @@ class GladeInput<T> {
   /// Can be dirty or pure.
   bool get isUnchanged => valueComparator?.call(initialValue, value) ?? _valueIsSameAsInitialValue;
 
-  /// Input does not have conversion error nor validation error.
+  /// Input does not have conversion error nor validation errors but can icnlude warnings.
   bool get isValid => !hasConversionError && validatorInstance.validate(value).isValid;
 
-  /// True when input is not valid.
+  bool get isValidWithoutWarnings => !hasConversionError && validatorInstance.validate(value).isValidWithoutWarnings;
```

Comment:
A bit hard to read to me. Naming alternatives: `isValidAndWithoutWarnings`, `isStrictlyValid`, `isValidNoWarnings`, ... ?